.circleci/config.yml

```diff
@@ -12,68 +12,68 @@ jobs:
     <<: *plain-go116
 
     steps:
-    - checkout
-
-    # -mod=readonly is enabled by default starting in go1.16 causing many go
-    # tools and editor integrations to report problems if the go.sum file is
-    # not up-to-date, so let's make sure it is.
-    - run:
-        name: go.sum up-to-date
-        command: |
-          if ! go run -mod=readonly gosum.go; then
-            # Older go versions, e.g. go1.14 will fail the check above with
-            # the message below.
-            #
-            #   go: updates to go.sum needed, disabled by -mod=readonly
-            #
-            # Newer versions show which go.sum entries are missing. To get
-            # useful CI errors for older versions, always print an explicit
-            # diff when go.sum is not up-to-date.
-            cp go.sum go.sum.before
-            go mod tidy
-            echo "--> go.sum diff:"
-            diff go.sum.before go.sum
-            exit 1
-          fi
-    - run:
-        name: milestone
-        command: |
-          go run checkmilestone.go
-
-    - run:
-        name: copyright
-        command: |
-          go run checkcopyright.go
+      - checkout
```

Comment:
❓ are these whitespace changes unrelated?

Reply:
The file isn't properly formatted yes. 
But I reverted them.